### PR TITLE
protocol/docs: revised protocol roadmap

### PR DESCRIPTION
**Pull request by danrobinson**
Edited language in Protocol Roadmap paper.

### CODE REVIEW

**Comment by danrobinson** [comment]
PTAL @oleganza

**Comment by oleganza** [comment]
@danrobinson  Unless the list has super-short items, I prefer sentences to end with a period. It's weird for the eye to not hit the period after reading a whole line. What do you say?

File: docs/protocol/papers/protocol-roadmap.md [hunk 1]
```diff
@@ -8,23 +8,23 @@ If you have feature requests or feedback, we want to hear from you. Join us on [
 
 ## Denial of service mitigation
 
-* Explicit limits on number and size of blockchain entities (size of the blocks, number of transactions etc).
-* Fine-tuned runtime cost limits for the control and issuance programs.
-* Improvements to consensus algorithm.
+* Fine-tuned and tested runtime costs for the virtual machine
```
Comment (oleganza) [inline docs/protocol/papers/protocol-roadmap.md]: Unless the list has super-short items, I prefer sentences to end with a period. It's weird for the eye to not hit the period after reading a whole line.

File: docs/protocol/papers/protocol-roadmap.md [hunk 2]
```diff
@@ -8,23 +8,23 @@ If you have feature requests or feedback, we want to hear from you. Join us on [
...
+* Adjustable run limit, as well as adjustable limits on the size of blocks, transactions, programs, and other data structures
+* Improved consensus algorithms to guarantee liveness under weaker assumptions
 
 ## Privacy
 
-* Homomorphically encrypted asset identifiers and amounts to provide secrecy for balances and financial parameters of the transactions.
-* Controlled traceability of the transactions; hiding the link between transaction inputs and the previous transactions’ outputs.
+* Confidential transactions that hide assets and amounts through homomorphic encryption
+* Unlinkable transactions that further hide the identities of transacting parties
```
Comment (oleganza) [inline docs/protocol/papers/protocol-roadmap.md]: Not unlinkable, but untraceable. Unlinkability refers to links between outputs and accounts. I'd like to keep using these terms consistently with existing research.

File: docs/protocol/papers/protocol-roadmap.md (hunk 1 above)
Comment (danrobinson) [inline docs/protocol/papers/protocol-roadmap.md]: They're not sentences, though; they're fragments.

Comment (oleganza) [inline docs/protocol/papers/protocol-roadmap.md]: Ok, i'm convinced. Thanks.

**Comment by oleganza** [comment]
LGTM

**Pull request merged**